src/ibek/ioc.py

```diff
@@ -122,14 +124,15 @@ def make_entity_classes(support: Support) -> types.SimpleNamespace:
     setattr(modules, support.module, module)
     modules.__all__.append(support.module)
     for definition in support.definitions:
-        entity_cls = make_entity_class(definition, support)
-        setattr(module, definition.name, entity_cls)
+        id_to_entity = make_entity_class(definition, support)
+        setattr(module, definition.name, id_to_entity)
```

Comment:
The old name made more sense

```suggestion
        entity_cls = make_entity_class(definition, support)
        setattr(module, definition.name, entity_cls)
```

Reply:
just here, like you've done